src/link/object.c

```diff
@@ -519,7 +519,7 @@ void obj_ReadFile(char const *fileName, unsigned int fileID)
 	tryReadlong(revNum, file, "%s: Cannot read revision number: %s",
 		    fileName);
 	if (revNum != RGBDS_OBJECT_REV)
-		errx("%s is a revision 0x%04" PRIx32 " object file; only 0x%04x is supported",
+		errx("%s is a revision 0x%04" PRIx32 " object file; only 0x%04x is supported. Try re-assembling this file or ensuring rgbasm and rgblink are up to date.",
```

Comment:
It would be more useful to at least report RGBLINK's version, and tell the user to re-build the object file with an updated RGBASM or RGBLINK, depending on which revision number is higher.

Reply:
Fixed

Reply:
I think it should only mention assembling if RGBASM's version is lower.

Reply:
Not necessarily, someone might cloning a repository with leftover object files from a more recent format. Re-assembling would still fix that.

Reply:
Then "try reassembling the file with RGBASM v<RGB.LI.NK>?"